test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -216,14 +216,10 @@ test.describe( 'Image', () => {
 		await expect( image ).toBeVisible();
 		await expect( image ).toHaveAttribute( 'src', new RegExp( fileName ) );
 
-		// Navigate to More,
-		await pageUtils.pressKeyWithModifier( 'shift', 'Tab' );
-		// Link,
-		await pageUtils.pressKeyWithModifier( 'shift', 'Tab' );
-		// Italic,
-		await pageUtils.pressKeyWithModifier( 'shift', 'Tab' );
-		// and finally Bold.
-		await pageUtils.pressKeyWithModifier( 'shift', 'Tab' );
+		// Navigate to Image.
```

Comment:
I'm not honestly sure how this ever worked. This needs bad accessibility help. Had no idea a double toolbar could exist.